src/coverlet.core/Reporters/JsonReporter.cs

```diff
@@ -1,22 +1,26 @@
 ﻿// Copyright (c) Toni Solarin-Sodara
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Text.Json;
```

Comment:
I activated option `JavaScriptEncoder.UnsafeRelaxedJsonEscaping`. We use only **XML file** or **console output** and not a **payload** for the coverage results.

>Unlike the [Default](https://learn.microsoft.com/en-us/dotnet/api/system.text.encodings.web.javascriptencoder.default?view=net-7.0#system-text-encodings-web-javascriptencoder-default) encoder, this encoder instance does not escape HTML-sensitive characters such as <, >, &. As a result, it must be used cautiously; for example, it can be used if the output data is within a response whose content-type is known with a charset set to UTF-8.

>:heavy_exclamation_mark: Compared to the default encoder, the UnsafeRelaxedJsonEscaping encoder is more permissive about allowing characters to pass through unescaped: 
>
>- It doesn't escape HTML-sensitive characters such as <, >, &, and '.
>- It doesn't offer any additional defense-in-depth protections against XSS or information disclosure attacks, such as those which might result from the client and server disagreeing on the charset.
>
>Use the unsafe encoder only when it's known that the client will be interpreting the resulting payload as UTF-8 encoded JSON. For example, you can use it if the server is sending the response header Content-Type: application/json; charset=utf-8. Never allow the raw UnsafeRelaxedJsonEscaping output to be emitted into an HTML page or a <script> element.